src/contributors/yusufalp.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "Yusuf Alp", 
+  "desc": "Front end web developer who enjoys puzzles and coding.",
+  "github" : "https://github.com/yusufalp",
+  "twitter" : "https://twitter.com/yalp122", 
+  "linkedin" : "https://www.linkedin.com/in/yusuf-alp/"
```

```suggestion
  "linkedin": "https://www.linkedin.com/in/yusuf-alp/"
```
Remove unwanted `white-space`.